routes/blogy.js

```diff
@@ -1,40 +1,39 @@
 const router = require("express").Router();
-const blog = require("../backend_model/blog_post");
-const blogcomment = require("../backend_model/Comment_post");
+const blog = require("../model/post");
+const blogcomment = require("../model/comment");
 
 // For blog post
-router.post("/blog", async (req, res) => {
+router.post("/blogs", async (req, res) => {
   const { title, content, author } = req.body;
   console.info(req.body);
   const newBlog = new blog({ title, content, author });
 
 //   try and catch
   try {
       await newBlog.save()
-      res.send("blog saved")
+      res.send("Blog saved!")
```

Comment:
It might be better to send the blog that was just created instead of sending a text